net.tabletopclub.TabletopClub.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "net.tabletopclub.TabletopClub",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tabletop-club-wrapper",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-documents/TabletopClub:create"
+    ],
+    "modules": [
+        {
+            "name": "tabletop-club",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D -m 755 -p TabletopClub.x86_64 /app/bin/tabletop-club",
```

Comment:
couldn't the Godot runtime that is available as a base app be used to provide this for  aarch64?

Reply:
Thank you for reviewing this PR so quickly! Due to the way the game imports custom assets from the file system, I had to make a custom fork of Godot, so I cannot use the runtime available in the repository. Reasoning is further explained in [the documentation](https://docs.tabletopclub.net/en/stable/general/download/compiling_from_source.html#why-did-you-fork-godot).

When this version of Tabletop Club (v0.1.2) was released, aarch64 was not yet supported in Godot 3.4.5, but once my game uses a more up-to-date version of Godot in the first major update, v0.2.0, I plan to investigate how I can compile my fork so that this architecture is supported. If I am successful, I shall update this repository to support it.

I shall push a commit for removing the SHA512 line, but do you want me to change this section as well?

Reply:
then maybe you could build it from source here so that aarch64 can be supported?

Reply:
Godot 3.4.5, the version of Godot that is used in this version of Tabletop Club, does not have support to be compiled for that architecture, unfortunately. I have tested this version of the game with Godot 3.5.x which I believe does have support for it now, but changes in the engine break the physics aspect of the game.

I use two particular repositories when compiling these specific versions of Godot, [godot-build-containers](https://github.com/drwhut/godot-build-containers) and [godot-build-scripts](https://github.com/drwhut/godot-build-scripts) - as far as I'm aware, aarch64 support was added for Godot 3.5 (which I'm planning to use for v0.2.0), and for Godot 4.x, but they did not bring support for Godot 3.4.x, as it was too old.

I would prefer to use my existing toolchain in order to build Tabletop Club releases, but if it becomes an obstacle to approve this repository then I will consider compiling it from scratch here.

Either that, or I can wait until the base game supports that architecture before updating this PR if that is preferred? Is it possible to add aarch64 support after the app has been approved and committed to Flathub?